apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -53,24 +75,27 @@ const ReviewApplication = ({ issue, requestIndex, readOnly }) => {
   const onAccept = () => onReviewApplication(true)
   const onReject = () => onReviewApplication(false)
   const changeRequest = (index) => setIndex(index)
+  const changeRequestType = (index) => {
+    setIndex(0)
+    setIndexType(index)
```

Comment:
I'm really digging my implicit suggestion above to rename `indexType` to `selectedFilter`. I think this block would end up being:

```js
const selectFilter = filter => {
  setIndex(0) // gotta come up with a better name for this, too!
  setSelectedFilter(filter)
}

Reply:
Renamed.